modules/diskq/logqueue-disk-reliable.c

```diff
@@ -332,13 +315,22 @@ _push_tail(LogQueue *s, LogMessage *msg, const LogPathOptions *path_options)
   gint64 message_position = qdisk_get_next_tail_position(self->super.qdisk);
   if (!qdisk_push_tail(self->super.qdisk, serialized_msg))
     {
+      EVTTAG *suggestion = NULL;
+      if (path_options->flow_control_requested)
+        {
+          suggestion = evt_tag_str("suggestion", "consider increasing mem-buf-size() or decreasing log-iw-size() "
+                                   "values on the source side to avoid message loss");
```

Comment:
I think it's very cool to make these suggestions! Is possible to make more suggestions elsewhere?

Reply:
AFAIK this is the only remaining part where we can lose messages by a log-fifo-size() misconfiguration.
For example, we don't drop flow-controlled messages anymore in LogQueueFifo.